test/suite/functional/Generator/ServiceProvider/ServiceProviderPhp70.php

```diff
@@ -29,7 +30,7 @@ class ServiceProvider
     public function register(Container $container): void
     {
         $container[BodySerializer::class] = static function (): BodySerializer {
-            return (new BodySerializer())->add('application/json', new JsonContentTypeSerializer())->add('application/x-www-form-urlencoded', new FormUrlencodedContentTypeSerializer());
+            return (new BodySerializer())->add(new JsonContentTypeSerializer())->add(new FormUrlencodedContentTypeSerializer())->add(new XmlContentTypeSerializer());
```

Comment:
hmm this is ugly, why cs fixer doesn't fix this? 🤔 maybe we miss some rule?

Reply:
I don't know, can't find anything for this.